src/main.go

```diff
@@ -221,6 +281,9 @@ func main() {
 			log.Infoln("initialized journal-metrics...", time.Since(p))
 			chainSet.Register(chain.Name, chain)
 
+			nullEncodedData := "0|0|0|0|0|0"
+			addNullBlock(chain, journalMetricsPath, nullEncodedData, "journal")
```

Comment:
Having addNullBLock in tsdb will make life simpler by not caring about type, since as the chain is formed, it should handle the types and other related information for null values. That way, its much easier to use, than always keeping in mind to call an addNullBlock() that does this, and passing all the necessary args.